test/unit/services/area-service.spec.ts

```diff
@@ -0,0 +1,97 @@
+import { expect } from 'chai';
+import nock from 'nock';
+import config from 'config';
+import logger from 'logger';
+import AreaService from 'services/areaService'
+import { createMockArea } from '../../e2e/utils/mock';
+import { getUUID } from '../../e2e/utils/helpers';
+import { ADMIN0_ISO, ADMIN2_ADMIN } from '../utils/test-constants';
+import { getTestServer } from '../../e2e/utils/test-server';
+
+describe('AreaService', () => {
+    before(async () => {
+        if (process.env.NODE_ENV !== 'test') {
+            throw Error(`Running the test suite with NODE_ENV ${process.env.NODE_ENV} may result in permanent data loss. Please use NODE_ENV=test.`);
+        }
+        
+        await getTestServer();
+    
+    });
+
+    afterEach(() => {
+        process.removeAllListeners('unhandledRejection');
+
+        if (!nock.isDone()) {
+            throw new Error(`Not all nock interceptors were used: ${nock.pendingMocks()}`);
+        }
+    });
+
+    describe('getUserArea', () => {
+        it('should fetch area data and return attributes using nock mock', async () => {
+            const areaId = getUUID();
+            const iso = { country: 'KE', region: 'Kiambu', subregion: 'Thika' };
+
+            createMockArea(areaId, iso);
+
+            const area = await AreaService.getUserArea(areaId);
+
+            logger.info(`area ${area}`)
```

Comment:
Do you want to keep all the `logger` statements in the tests? Or was it just an oversight?